ecs-cli/modules/clients/aws/cloudformation/template.go

```diff
@@ -207,13 +207,18 @@ var template = `
       "Type": "String",
       "Description": "Optional - Whether to create resources only for running Fargate tasks.",
       "Default": "false"
+    },
+    "UserData" : {
+      "Type" : "String",
+      "Description" : "User data for EC2 instances. Required for EC2 launch type, ignored with Fargate",
+      "Default" : ""
```

Comment:
Yeah, it wasn't, but after I fix the other comment you made then it will be. So what happens now is that the user data is always a MIME Multipart archive. If the user doesn't specify any extra user data files, then we just get a MIME Multipart with just the joining cluster user data.